src/views/polymerday-sponsors.html

```diff
@@ -44,11 +44,17 @@
     .title {
       margin-top: 0;
       margin-bottom: 16px;
+      color: var(--color-blue-primary);
     }
 
     .sponsors {
       max-width: 80%;
     }
+
+    .item-subtitle {
+      color: var(--color-grey-primary);
```

Comment:
Then we need a new style in the theme for this cases. @kcmr what do you recommend?